lib/cache.js

```diff
@@ -86,9 +86,13 @@ function getOpportunities() {
 }
 
 const cache = `
-  export const posts = ${getProjects()};
-  export const authors = ${getAuthors()};
-  export const opportunities = ${getOpportunities()};
+const postsWithoutId = ${getProjects()};
+export const posts = postsWithoutId.map(post => {
```

Comment:
`Math.random()` would assign a random id to this right? could we also add maybe something that would tell us what post it is like project name or something so? not too sure if it matters but might be more descriptive than a random number